docs/Project-Board-Guide.md

```diff
@@ -17,13 +17,13 @@ Please familiarise yourself with our:
 | Ready for Dev | Contains issues that have been refined. | Once you have selected an issue to work on, you must assign yourself to it and manually move the issue to the "Issues in progress".| n/a |
 | Externally blocked | Contains issues/PRs that are blocked by an external factor. | Manually move the issue/PR when it is no longer blocked. | n/a |
 | Issues in progress | Contains issues that are in working progress. | If the PR that uses a [keyword to link the issue](https://help.github.com/en/articles/closing-issues-using-keywords) has been merged, then the cards in this column will move automatically to "Done". | PRs should not use keywords unless they definitely complete the entire issue. |
-| PR in progress | Contains cards that have been automatically added here once a PR is assigned to the project "Simorgh". | It is your responsibility to move the card to "Code Review", when you wish for it to be reviewed by others. Teams prioritise code reviews over actual development. | PRs should be created on first commit, so that the team can see what you are working on. |
-| Code review | Contains PRs that are deemed ready to be reviewed by other developers. | Once 2 other developers have approved your PR, it will automatically be moved to "Ready for test", however if your PR requires "UX, A11Y, External Review", then you will have to manually move it there. | If you had paired with someone on an issue, then one participant can be the reviewer but the 2nd reviewer must be a code maintainer. |
-| UX, A11Y, External Review | Contains issues that are in progress of an accessibility swarm, UX or External review. | When the reviewing party approves the PR, the card will then have to be manually moved to "Ready for test". | [Read more about accessibility swarm](https://bbc-news.github.io/accessibility-news-and-you/accessibility-swarms) |
-| Ready for test | Contains PRs that are ready for the test engineers to test. | Once approved by a test engineer, it is their responsibility to update the PR with an approval and then the card will move to "Ready for merge". | n/a |
-| Ready for merge | Contains tickets that are approved by a test engineer. | If your PR has no CI (continuous integration) errors and the merge button is not disabled, then you can merge your PR and then it will automatically be moved to "Done". For cleanliness, you must also delete your branch after clicking merge. | n/a |
+| PR in progress | Contains cards that have been automatically added here once a PR is assigned to the project "Simorgh". | It is your responsibility to move the card to "Code Review", when you wish for it to be reviewed by others. Teams prioritise code reviews over actual development. | PRs should be created on first commit, so that the team can see what you are working on. Draft PRs are preferable until the code is ready for review, when it can be converted to a proper PR. |
```

Comment:
```suggestion
| PR in progress | Contains cards that have been automatically added here once a PR is assigned to the project "Simorgh". | It is your responsibility to move the card to "Code Review", when you wish for it to be reviewed by others. Teams prioritise code reviews over actual development. | Draft PRs should be created on first commit, so that the team can see what you are working on. Once the code is ready for review, it can be converted to a proper PR. |
```